src/main/java/roomescape/reservation/service/ReservationService.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.reservation.service;
+
+import java.util.List;
+import org.springframework.dao.DataAccessException;
+import org.springframework.stereotype.Service;
+import roomescape.reservation.dao.ReservationDao;
+import roomescape.reservation.domain.Reservation;
+import roomescape.reservation.dto.ReservationRequest;
+import roomescape.reservation.dto.ReservationResponse;
+import roomescape.time.dao.ReservationTimeDao;
+import roomescape.time.domain.ReservationTime;
+import roomescape.time.dto.TimeResponse;
+
+@Service
+public class ReservationService {
+    private final ReservationDao reservationDao;
+    private final ReservationTimeDao reservationTimeDao;
+
+    public ReservationService(final ReservationDao reservationDao, final ReservationTimeDao reservationTimeDao) {
+        this.reservationDao = reservationDao;
+        this.reservationTimeDao = reservationTimeDao;
+    }
+
+    public ReservationResponse createReservation(final ReservationRequest reservationRequest) {
+        try {
+            ReservationTime reservationTime = reservationTimeDao.findReservationTimeById(reservationRequest.timeId());
+            Reservation notSavedreservation = new Reservation(
+                    null, reservationRequest.name(), reservationRequest.date(), reservationTime
+            );
+            Reservation savedReservation = reservationDao.insertReservation(notSavedreservation);
+            return new ReservationResponse(
+                    savedReservation.getId(),
+                    savedReservation.getName(),
+                    savedReservation.getDate(),
+                    new TimeResponse(savedReservation.getTime().getId(), savedReservation.getTime().getStartAt())
+            );
+        } catch (DataAccessException dataAccessException) {
+            throw new IllegalArgumentException("[ERROR] 요청받은 timeId가 존재하지 않습니다.");
+        }
+    }
+
+    public void removeReservation(final long id) {
+        reservationDao.removeReservation(id);
+    }
+
+    public List<ReservationResponse>  findAllReservations() {
```

Comment:
```suggestion
    public List<ReservationResponse> findAllReservations() {
```

불필요한 공백이 있네요. 매의 눈이죠? ㅎㅎ 👀 
intelliJ 의 포맷팅 기능을 활용하면 일관성 있는 포맷을 유지할 수 있습니다.

Reply:
헉, 감사합니다!